plutus-core/plutus-ir/test/PlutusIR/Transform/RewriteRules/Tests.hs

```diff
@@ -1,21 +1,37 @@
 module PlutusIR.Transform.RewriteRules.Tests where
 
 import PlutusCore.Quote
+import PlutusCore.Test hiding (ppCatch)
+import PlutusIR.Compiler qualified as PIR
 import PlutusIR.Parser
 import PlutusIR.Test
 import PlutusIR.Transform.RewriteRules as RewriteRules
+import PlutusPrelude
 
-import Data.Default.Class
 import Test.Tasty
-import Test.Tasty.Extras
 
 test_RewriteRules :: TestTree
 test_RewriteRules = runTestNestedIn ["plutus-ir/test/PlutusIR/Transform"] $
     testNested "RewriteRules" $
-    fmap
-        (goldenPir (runQuote . RewriteRules.rewriteWith def)  pTerm)
-        [ "equalsInt" -- this tests that the function works on equalInteger
-        , "divideInt" -- this tests that the function excludes not commutative functions
-        , "multiplyInt" -- this tests that the function works on multiplyInteger
-        , "let" -- this tests that it works in the subterms
+    (fmap
+        (goldenPirDoc (prettyPlcClassicDebug . runQuote . RewriteRules.rewriteWith def)  pTerm)
+        [ "equalsInt.pir" -- this tests that the function works on equalInteger
```

Comment:
This is now inconsistent with all the other test files, which don't have the `.pir` suffix.